flow/project.py

```diff
@@ -2058,6 +2069,11 @@ def _add_print_status_args(cls, parser):
             nargs='*',
             help="Display select parameters of the job's "
                  "statepoint with the detailed view.")
+        view_group.add_argument(
+            '-l', '--all-parameters',
+            action='store_true',
+            help="Display all parameters of the job's "
+                  "statepoint with the detailed view.")
```

Comment:
I would replace `job's` with `jobs'` to indicate that we are showing the parameters for all jobs, not just of one.